.eslintrc.json

```diff
@@ -1,9 +1,5 @@
 {
     "extends": "airbnb",
-    "plugins": [
```

Comment:
checkout https://www.npmjs.com/package/eslint-config-airbnb for installing these plugins